server_addon/maya/server/settings/creators.py

```diff
@@ -373,6 +381,18 @@ class CreatorsModel(BaseSettingsModel):
             "Main"
         ]
     },
+    "CreateOxRig": {
+        "enabled": True,
+        "default_variants": [
+            "Main"
+        ]
+    },
+    "CreateOxCache": {
+        "enabled": True,
+        "default_variants": [
+            "Main"
+        ]
+    },
```

Comment:
Should these default to `False` so that only studios using Ornatrix could enable them?